src/iotconnect_telemetry.c

```diff
@@ -252,13 +264,21 @@ bool iotcl_telemetry_set_null(IotclMessageHandle message, const char *path) {
 }
 
 const char *iotcl_create_serialized_string(IotclMessageHandle message, bool pretty) {
-    if (!message) return NULL;
-    if (!message->root_value) return NULL;
-    if (pretty) {
-        return cJSON_Print(message->root_value);
-    } else {
-        return cJSON_PrintUnformatted(message->root_value);
+    const char *serialized_string = NULL;
+    if (!message) {
+        PRINTF("message is NULL\n");
```

Comment:
are these PRINTFs a mistake? The library should not be printing anything, though I don't mind adding optional debug print support.

Reply:
They are a mistake -- well spotted :-)